server/controllers/BookController.js

```diff
@@ -75,6 +77,42 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+ * Add a new book
+ * @name addBook
+ * @param {Object} request
+ * @param {Object} response
+ * @returns {JSON} object
+ */
+  static async getBorrowingStats(request, response) {
+    try {
+      const user = JSON.parse(JSON.stringify(request.userData));
+      const { id } = user;
+      let filter = { userId: id };
+      const { roleName } = user.UserRoles[0].Role;
+      if (roleName === 'admin' || roleName === 'superadmin') filter = {};
+      const { limit, page } = request.query;
+      const { count } = await LendingHistory.findAndCountAll({ where: filter });
+      const totalPages = Math.ceil(count / (limit || 10));
+      if ((page || 1) > totalPages) return responseMessage(response, 400, { status: 'failure', message: 'no book(s) found on this page' });
+      const offset = (limit || 10) * ((page || 1) - 1);
```

Comment:
instead of always repeat limit || 10 and page || 1, why not declare it once, some you don,t have to repeat yourself each time you want to make use of it.

Reply:
I'm only refactoring the code for the function does not exit 25 lines